test/ecto/query/inspect_test.exs

```diff
@@ -268,15 +268,29 @@ defmodule Ecto.Query.InspectTest do
            ~s{from p0 in Inspect.Post, lock: "FOOBAR"}
   end
 
-  test "preload" do
-    assert i(from(x in Post, preload: :comments)) ==
-           ~s"from p0 in Inspect.Post, preload: [:comments]"
+  # TODO: AST is represented as string differently on versions pre 1.13
+  if Version.match?(System.version(), ">= 1.13.0-dev") do
```

Comment:
I'm not 100% sure if this one is the same as the others. It is removing the brackets around the keyword list